src/main/java/cz/muni/fi/pa165/bluebat/dao/TripDao.java

```diff
@@ -0,0 +1,40 @@
+package cz.muni.fi.pa165.bluebat.dao;
+
+import cz.muni.fi.pa165.bluebat.entity.Trip;
+
+/**
+ * @author Ondrej Vaca
+ * An interface for manipulation with the Trip database table.
+ */
+
+public interface TripDao {
```

Comment:
Ještě bych přidal funkci findAll(), která by vrátila všechno, To si myslím bude dobré, když budeme chtít zobrazit všechny na jedné stránce pro zákazníky